catalog/nsili/catalog-nsili-endpoint/src/test/java/org/codice/alliance/nsili/endpoint/OrderRequestImplTest.java

```diff
@@ -469,7 +501,7 @@ protected void writeFile(FileLocation destination, InputStream fileData, long si
         assertThat(holder.value.elements[0].files.length, is(1));
     }
 
-    @Test(expected = NO_IMPLEMENT.class)
+    @Test/*(expected = BAD_OPERATION.class)*/
```

Comment:
getBadDestination should probably be changed to h_dest as we don't support physical delivery and the method should be re-verified that it will throw a BAD_OPERATION exception.